app/.env.example

```
@@ -1,3 +1,4 @@
+Environment=test
 SUPABASE_URL=https://supabase_url.supabase.co
 SUPABASE_KEY=SUPABASE_SERVIVCE_KEY
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SUPABASE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)